cosmic_ray/worker.py

```diff
@@ -28,18 +27,20 @@
 from .parsing import get_ast
 from .testing.test_runner import TestOutcome
 from .work_item import WorkItem, WorkItemJsonDecoder
+import cosmic_ray.compat.json
```

Comment:
The mix of relative and absolute import here is a bit jarring. FWIW I have a strong preference for absolute imports. They just make me think less. Explicit is better that implicit, yada yada